wire-tests-swift/no-manifest/src/main/swift/AllSwiftWrappers.swift

```diff
@@ -0,0 +1,609 @@
+        self.storage = try container.decode(Storage.self)
+    }
+
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.singleValueContainer()
+        try container.encode(storage)
+    }
+
+}
+#endif
+
+extension AllSwiftWrappers {
+
+    /**
+     * Underlying storage for AllSwiftWrappers
+     */
+    public struct Storage {
+
+        public var double_value: Double?
+        public var float_value: Float?
+        public var int64_value: Int64?
+        public var uint64_value: UInt64?
+        public var int32_value: Int32?
+        public var uint32_value: UInt32?
+        public var bool_value: Bool?
+        public var string_value: String?
+        public var bytes_value: Foundation.Data?
+        public var rep_double_value: [Double] = []
+        public var rep_float_value: [Float] = []
+        public var rep_int64_value: [Int64] = []
+        public var rep_uint64_value: [UInt64] = []
+        public var rep_int32_value: [Int32] = []
+        public var rep_uint32_value: [UInt32] = []
+        public var rep_bool_value: [Bool] = []
+        public var rep_string_value: [String] = []
+        public var rep_bytes_value: [Foundation.Data] = []
+        public var map_int32_double_value: [Int32 : Double] = [:]
+        public var map_int32_float_value: [Int32 : Float] = [:]
+        public var map_int32_int64_value: [Int32 : Int64] = [:]
+        public var map_int32_uint64_value: [Int32 : UInt64] = [:]
+        public var map_int32_int32_value: [Int32 : Int32] = [:]
+        public var map_int32_uint32_value: [Int32 : UInt32] = [:]
+        public var map_int32_bool_value: [Int32 : Bool] = [:]
+        public var map_int32_string_value: [Int32 : String] = [:]
+        public var map_int32_bytes_value: [Int32 : Foundation.Data] = [:]
+        public var unknownFields: UnknownFields = .init()
+
+        public init(configure: (inout Self) -> Swift.Void = { _ in }) {
+            configure(&self)
+        }
+
+    }
+
+}
+
+#if !WIRE_REMOVE_EQUATABLE
+extension AllSwiftWrappers.Storage : Equatable {
+}
+#endif
+
+#if !WIRE_REMOVE_HASHABLE
+extension AllSwiftWrappers.Storage : Hashable {
+}
+#endif
+
+extension AllSwiftWrappers.Storage : Sendable {
+}
+
+extension AllSwiftWrappers.Storage : ProtoMessage {
+
+    public static func protoMessageTypeURL() -> String {
+        return "type.googleapis.com/squareup.proto3.kotlin.swift_all_wrappers.AllSwiftWrappers"
+    }
+
+}
+
+extension AllSwiftWrappers.Storage : Proto2Codable {
+
+    public init(from protoReader: ProtoReader) throws {
+        var double_value: Double? = nil
+        var float_value: Float? = nil
+        var int64_value: Int64? = nil
+        var uint64_value: UInt64? = nil
+        var int32_value: Int32? = nil
+        var uint32_value: UInt32? = nil
+        var bool_value: Bool? = nil
+        var string_value: String? = nil
+        var bytes_value: Foundation.Data? = nil
+        var rep_double_value: [Double] = []
```

Comment:
hmmm @efirestone I suspect this needs to be `[Double?]` if that is what these Google wrapper types were made for right?